makelib/common.mk

```diff
@@ -187,7 +187,7 @@ ifeq ($(origin VERSION), undefined)
 # check if there are any existing `git tag` values
 ifeq ($(shell git tag),)
 # no tags found - default to initial tag `v0.0.0`
-VERSION := $(shell echo "v0.0.0-$$(git rev-list HEAD --count)-$$(git describe --dirty --always)" | sed 's/-/./2' | sed 's/-/./2')
```

Comment:
i validated that when there are no tags in a repo, the only thing generated from `git describe --dirty --always` is the commit short hash without the leading `g`, so this change does seem valid. I was worried it might create a duplicated leading `g`, but it does not seem to :)